protecode/model.py

```diff
@@ -25,6 +25,11 @@ class ProcessingStatus(Enum):
     FAILED = 'F'
 
 
+class CVSSVersion(Enum):
+    V2 = "V2"
```

Comment:
according to my research, CVSS versions are written with lower-cased v. also, please use `'` to enclose str literals for consistency reasons

it could be considered to use `CVSSv2` `CVSSv3`, which I also found e.g. on wikipedia / other sources